src/css/_youtube.scss

```diff
@@ -67,18 +67,10 @@
   }
   &--gradient-fill {
     position: absolute;
-    left: 0;
-    margin-left: -80px;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    width: 1500px;
+    width: 100vw;
     height: 540px;
     opacity: 0.53;
     background-image: radial-gradient(circle at 23% 54%, rgba(40, 40, 40, 0.5), rgba(40, 40, 40, 0));
```

Comment:
We're affecting the opacity with the opacity rule as well as the rgba in the gradient, wonder if instead of lines 72 & 73 we could just do:

`
background-image: radial-gradient(circle at 23% 54%, rgba(40, 40, 40, 0.2), rgba(40, 40, 40, 0));
`